go.mod

```diff
@@ -33,8 +33,8 @@ require (
 	github.com/sarslanhan/cronmask v0.0.0-20190709075623-766eca24d011
 	github.com/sirupsen/logrus v1.8.1
 	github.com/sony/gobreaker v0.5.0
-	github.com/stretchr/testify v1.8.0
-	github.com/szuecs/rate-limit-buffer v0.7.1
+	github.com/stretchr/testify v1.7.0
+	github.com/szuecs/rate-limit-buffer v0.7.3-0.20221230130449-bee752deeb50
```

Comment:
Should we perform lib release to get v0.7.3 maybe?

Reply:
I think yes we should even increase the minor version because of the breaking changes, @szuecs wdyt?

Reply:
v0.8.0 here you go